### PR TITLE
feat(Metabase): collection inheritance and integration improvements with BigQuery

### PR DESCRIPTION
**Pull request by PatrickfBraz**
## Context and Motivation
Metabase ingestion is functional and meets most of the cases. However, for the use case of a company I work for, it wasn't working properly. The behavior of extracting information when the engine was BigQuery did not work as expected. It was impossible to create data lineage between what was entered from the BigQuery ingestion and the information provided by the Metabase ingestion. In this sense, the objective of this PR is to improve the ingestion when the data source is BigQuery and to bring new functionality to create Containers based on Metabase collections.

## Relevant remarks

* In creating the platform, the official logo of the Metabase found at `https://www.metabase.com/images/logo.svg` was used  
* The previous ingestion code did not consider cases where a card can be sourced from one or more cards. In this sense, the method that extracts the lineage does a recursive search to find the source tables.
* The previous code did not consider that in the case of BigQuery the paths to the tables were given by <project_id>.<dataset_id>.<table_id> and therefore it was not possible to build the lineage. Now the project_id is considered in the construction of the Datasets URNs

### Some images to demonstrate the results
It is now possible to create a Container based on Collections
image
The container inherits the information from the collection
image
Can now establish lineage with BigQuery tables inserted by the ingestion provided by the DataHub itself (backward compatibility)
image

## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [x] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: metadata-ingestion/source_docs/metabase.md [hunk 1]
```diff
@@ -29,6 +29,15 @@ retrieve the following dashboard information.
 - Owner
 - Link to the dashboard in Metabase
 - Associated charts
+- Dashboard parameters
+
+The following properties for a Darshboard are ingested in DataHub.
```
Comment (viniciusdsmello) [inline metadata-ingestion/source_docs/metabase.md:34]:
```suggestion
The following properties for a Dashboard are ingested in DataHub.
```

**Comment by maggiehays** [comment]
@Mohd-gslab Can you take a look at this one, please?

**Comment by oristides** [comment]
@PatrickfBraz Why did you close?

**Comment by PatrickfBraz** [comment]
I closed this pull request because of the reasons:

1. Metadata ingestion framework has several updates. This code was implemented for version 0.8.37 and an older version of Metabase
2. I'm working on a more significant improvement tested in my company's deployment.

In our use case, the data source for Metabase is BigQuery, and the actual version doesn't deal well with it. Because of that, we are planning to open a pull request with our improvements. Maybe next months I will open other pull request.

**Pull request closed without merging**